app/domain/errors.rb

```diff
@@ -446,12 +446,12 @@ module AuthnJwt
       )
 
       UnsupportedAuthenticator = ::Util::TrackableErrorClass.new(
-        msg: "Authenticator '{0-authenticator-name}' is unsupported.",
```

Comment:
@Avi-Eyal  what is the reason for this change?

Reply:
There is no particular reason, the convention in the other logs is that where there is only one parameter or the specific parameter is self explained then you dont need to specify the parameter by name.

personally I dont really care to rollback my changes for this line